pages/index.css

```diff
@@ -26,3 +26,14 @@
 .color-white {
     color: white;
 }
+
+.you-break-we-monitor {
```

Comment:
should be `you-brake-we-monitor` not `break`, also will need to be adjusted in other places :)

Reply:
😅 hahah good catch